.changeset/red-candles-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/sitemap': minor
+---
+
+Introduce option to add custom sitemaps to index
```

Comment:
Can you please provide more description? You're about to explain a new feature to Astro users, so you'd want to put more description about the feature (preferably beginner-friendly). Also, you'd want to explain what a "custom sitemap" is

Reply:
This feature is very similar to the [`customPages`](https://docs.astro.build/en/guides/integrations-guide/sitemap/#custompages) option but in the sense that it let's users include other sitemaps that could be hosted on a site into the `sitemap-index.xml` file that astro generates.

- `customPages`: They are included with the pages a site has and go along with them into `sitemap-${i}.xml` file.
- `customSitemaps`: They will be included with all the `sitemap-${i}.xml` files and go into `sitemap-${index}.xml` file.

I used the word "custom" because `customPages` exists, otherwise I think "external" could make sense too.

I will update the changeset entry and add all this info, would be great if you could point if this info needs more clarity? Thank you!

Reply:
In the changeset I could describe this scenario as I did in the PR:

> Ability to add `customSitemaps` is important because in my case I host multiple services under the same domain proxied by nginx: `example.com` (made with astro), `example.com/blog` (made with ghost, etc.). So I need to be able to add `example.com/blog/sitemap.xml` to the default `sitemap-index.xml` file.

Reply:
You want to explain how to use it (to me too, because I'm not familiar with `customPages`), use a case, and maybe a code snippet of the configuration that explains what's the end result.

Reply:
I've updated the changeset 🫡